PhysicsTools/PatAlgos/python/producersLayer1/muonProducer_cfi.py

```diff
@@ -132,7 +134,7 @@
 
 
 
-
+Run2_2016.toModify(patMuons, mvaTrainingFile="RecoMuon/MuonIdentification/data/mu_2017_BDTG.weights.xml")
```

Comment:
this is the same as the original.
what is the reason to add toModify?

Reply:
fixed in the latest commit... Also @bmahakud can you confirm whether this is the proper era modifier?